doc/configcommands.dsv

```diff
@@ -4,7 +4,8 @@ article-sort-order||<sortfield>[-<direction>]||date-asc||The <sortfield> specifi
 articlelist-format||<format>||"%4i %f %D %6L  %?T?|%-17T|  ?%t"||This variable defines the format of entries in the article list. See the respective section in the documentation for more information on format strings.||articlelist-format "%4i %f %D   %?T?|%-17T|  ?%t"
 articlelist-title-format||<format>||"%N %V - Articles in feed '%T' (%u unread, %t total)%?F? matching filter '%F'&? - %U" (localized)||Format of the title in article list. See "Format Strings" section of Newsboat manual for details on available formats.||articlelist-title-format "Articles in feed '%T' (%u unread)"
 auto-reload||[yes/no]||no||If set to `yes`, all feeds will be automatically reloaded at start up and then continuously after a certain time has passed (see <<reload-time,`reload-time`>>). See also <<refresh-on-startup,`refresh-on-startup`>> to only reload the feeds at start up, but not continuously. Enabling <<suppress-first-reload,`suppress-first-reload`>> omits the reload on start up.||auto-reload yes
-bind-key||<key> <operation> [<dialog>]||n/a||Bind key <key> to <operation>. This means that whenever <key> is pressed, then <operation> is executed (if applicable in the current dialog). For more information see <<_key_bindings>>. See also <<unbind-key,`unbind-key`>> to remove a key binding.||bind-key ^R reload-all
+bind||<key-sequence> <dialog>[,<dialog>] <command-list> [-- "<binding description>"]||n/a||Bind sequence of keys <key-sequence> to <command-list>. This means that whenever the keys in <key-sequence> are pressed in order, then the list of commands in <command-list> are executed (if applicable in the current dialog). For more information see <<_key_bindings>>. Optionally, a description can be added. If present, the description is shown in the help form. See also <<unbind-key,`unbind-key`>> to remove a key binding.||bind of everywhere set browser "firefox" ; open-in-browser
```

Comment:
> See also <<unbind-key,`unbind-key`>> to remove a key binding.

Oops, I totally forgot about unbinding! Does it actually work? The code in `Keymap::handle_action` uses `KeyCombination::from_bindkey` when handling `unbind-key`. It doesn't look like we have a test for this either.

Reply:
Yes, it does work.
It has a limitation in that it only allows specifying a single key (no sequence of keys).
However, given that the default configuration only contains single-key bindings, I don't see much of a reason for wanting to unbind a specific multi-key key-sequence.

Reply:
I agree this isn't release-blocking, but I still think it's important to have `unbind`. More on that in https://github.com/newsboat/newsboat/issues/3034.